ctru-rs/src/applets/mii_selector.rs

```diff
@@ -8,14 +8,14 @@ use std::ffi::CString;
 
 /// Index of a Mii used to configure some parameters of the Mii Selector
 /// Can be either a single index, or _all_ Miis
-#[derive(Debug, Clone)]
-pub enum MiiConfigIndex {
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+pub enum MiiIndex {
```

Comment:
this could probably even just be called `mi_selector::Index`, same for other structs in this module.

Or actually, we could call the module `mii` and then you'd have `mii::Type`, `mii::Selector`, etc.

Reply:
The mii module already exists, lol. I can rename it to settle for a better `module::struct` nomenclature.

Reply:
Yeah, `mii:selector` seems like a good option tbh

Reply:
That would move the module out of the `applets` folder. Do you think that would be a good idea? There are many more applets left to be implemented.

Reply:
I simply renamed it to `Index` for now. Tell me if you think more changes are needed.